lib/util/animations.js

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
Huh, isn't this also in PR #5?

Reply:
huh yeah I don't know how that happened...